.editorconfig

```diff
@@ -171,6 +171,9 @@ csharp_prefer_braces = true:silent
 
 dotnet_diagnostic.RS0041.severity = suggestion
 
+# RS1035: Do not use APIs banned for analyzers(such as Environment.NewLine)
+dotnet_diagnostic.RS1035.severity = suggestion
```

Comment:
Can we move this into a more scoped .editorconfig file so that we don't disable the banned API analyzer for other projects?

Reply:
That is certainly appropriate.

- T4 autogenerated Transformer files <- .editorconfig
- `MessagePack.SourceGenerator/MessagePack.SourceGenerator.Emit.cs` <- pragma warning